### PR TITLE
Add a task's activity to the data returned for a single task

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: lib/models/case.js [hunk 1]
```diff
@@ -135,6 +137,14 @@ class Case {
 
   static find(id) {
     return this.Model.query().findById(id)
+      .eager('[activityLog]')
+      .modifyEager('[activityLog]', builder => {
+        // we only care about logs of status changes
+        builder.where('eventName', 'like', 'status%')
+          // ignore auto-forwarded status changes
+          .andWhere('eventName', '!=', 'status:with-ntco:ntco-endorsed')
```
Comment (lennym) [inline lib/models/case.js]: Can we do that filtering at render-time? It's _super_ implementation specific, so doesn't really have any place in framework level code.

Comment (wheelsandcogs) [reply lib/models/case.js]: Done.

File: lib/activitylog/hooks/logger.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ module.exports = db => {
   return model => {
     return ActivityLog.query(db).insert({
       caseId: model.id,
-      changedBy: get(model, 'meta.user.id'),
+      changedBy: get(model, 'meta.user.profile.id'),
```
Comment (lennym) [inline lib/activitylog/hooks/logger.js]: How feasible is it to add a middleware in workflow that does `req.user = req.user.profile`?

I'm wary about leaking implementation detail (user object structure) into the framework, but also if it's in any way a pain then I'll live with this.

Comment (lennym) [reply lib/activitylog/hooks/logger.js]: It's a pain. I'll live with it.

**Pull request merged**